css/properties/touch-action.json

```diff
@@ -152,7 +152,7 @@
                 "version_added": false
               },
               "safari_ios": {
-                "version_added": "9.1"
+                "version_added": false
```

Comment:
Can we set the parent `safari_ios` to null instead of setting this one to false? I can't actually test this (no iOS device) but there seems to be lots of evidence (e.g., StackOverflow questions, the linked WebKit bug) that `touch-action` does _something_ on iOS